<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Checkers_Game</class>
 <widget class="QMainWindow" name="Checkers_Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1613</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Start_Game">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>40</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Start_Game {
    border-image: url(:/images/assets/button_none.png);
}

QPushButton#Start_Game:hover {
    border-image: url(:/images/assets/button_hover.png);
}

QPushButton#Start_Game:pressed {
    border-image: url(:/images/assets/button_press.png);
}
</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Setting">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>540</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>40</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Setting {
    border-image: url(:/images/assets/button_none.png);
}

QPushButton#Setting:hover {
    border-image: url(:/images/assets/button_hover.png);
}

QPushButton#Setting:pressed {
    border-image: url(:/images/assets/button_press.png);
}
</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>750</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Exit {
    border-image: url(:/images/assets/button_none.png);
}

QPushButton#Exit:hover {
    border-image: url(:/images/assets/button_hover.png);
}

QPushButton#Exit:pressed {
    border-image: url(:/images/assets/button_press.png);
}
</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="Help">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Help {
    border-image: url(:/images/assets/button_none.png);
}

QPushButton#Help:hover {
    border-image: url(:/images/assets/button_hover.png);
}

QPushButton#Help:pressed {
    border-image: url(:/images/assets/button_press.png);
}
</string>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1613</width>
      <height>907</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/assets/b.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>Start_Game</zorder>
   <zorder>Setting</zorder>
   <zorder>Exit</zorder>
   <zorder>Help</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1613</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>新游戏   </string>
     </property>
     <addaction name="action"/>
     <addaction name="separator"/>
     <addaction name="action_3"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menuSounds">
     <property name="title">
      <string>声音 </string>
     </property>
     <widget class="QMenu" name="menuBGM">
      <property name="title">
       <string>背景音乐</string>
      </property>
      <addaction name="actionON_2"/>
      <addaction name="separator"/>
      <addaction name="actionOFF_2"/>
     </widget>
     <widget class="QMenu" name="menuEffect">
      <property name="title">
       <string>音效</string>
      </property>
      <addaction name="actionON_3"/>
      <addaction name="separator"/>
      <addaction name="actionOFF_3"/>
     </widget>
     <addaction name="menuBGM"/>
     <addaction name="separator"/>
     <addaction name="menuEffect"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuSounds"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>教程</string>
    </property>
    <addaction name="actionOpen_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>加载存档</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>存档</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionRegret">
   <property name="text">
    <string>悔棋</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionON_2">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionOFF_2">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionON_3">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionOFF_3">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="actionbackground">
   <property name="text">
    <string>背景图片</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>本地对战</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>人机对战</string>
   </property>
  </action>
  <action name="actionOpen_Settings">
   <property name="text">
    <string>打开设置</string>
   </property>
  </action>
  <action name="actionOpen_Help">
   <property name="text">
    <string>打开教程</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
